settings/Multisite.setting.php

```diff
@@ -20,6 +20,7 @@
  */
 
 return [
+  // FIXME: This is arguably the worst name for a setting ever
   'is_enabled' => [
```

Comment:
@eileenmcnaughton maybe we could add a way to rename and alias settings? Probably trigger a deprecation when accessed by the old name.

Reply:
I believe Eileen did a rename/reformat a few years back for a setting called `contribution_invoice_settings` (which was apparently a pretty gnarly setting). A straight-up rename should be simpler, but the important thing is that it led to the methods `SettingsBag::updateVirtual()` and `SettingsBag::createVirtual()`. That would probably be the easiest place to inject an alias mechanism.

Reply:
@eileenmcnaughton I feel like its time to add noisy deprecation to `contribution_invoice_settings`